wrappers/msbuild

```diff
@@ -66,5 +66,8 @@ arm64)
     ;;
 esac
 
+# Prefer x86_64 build tools for any target arch.
+export PreferredToolArchitecture=x64
```

Comment:
Is it working on arm64 linux @mstorsjo

Reply:
Is there an arm64 port of Wine-Mono though? MSBuild is a .NET 4.x executable, so it needs Mono to run on Wine.

There are no arm64 Mono releases, at least here: https://github.com/madewokherd/wine-mono/releases/tag/wine-mono-9.0.0

Reply:
I haven’t tested anything with mono/msbuild yet, no. While there might not be any release of it, I presume it could be possible to build it (but I haven’t spent effort in trying to set that up).

So it think we should have the same logic here, as we have in install.sh, for picking the host executable arch - even if actually testing it on arm64 still is up in the air. (Keeping it pointing at x64 won’t work in any case, while pointing at arm64 probably should work out and be correct, even if there may be other challenges around mono on arm64.)

Reply:
> Keeping it pointing at x64 won’t work in any case, while pointing at arm64 probably should work out and be correct, even if there may be other challenges around mono on arm64.

I studied MSBuild VC property files some more, and it looks like arm64 tools are always preferred on arm64 hosts, so there is no need to set this property in that case.

I'll wrap it in
```sh
if [[ `uname -m` == "x86_64" ]];
```
just to be safe.